index.js

```diff
@@ -0,0 +1,152 @@
+// The Front object is loaded through the Front script added in the header of the main HTML.
+// This object can be used to listen to conversation event data as it occurs on Front, request information from Front, and perform actions on Front.
+// See the full plugin API documentation here: https://dev.frontapp.com/plugin.html
+
+// This keeps back of if Front has returned a conversation to the plugin.
+let hasConversation;
+
+// Listen for the `conversation` event from Front and print its contents, then load the contact to the plugin.
+Front.on('conversation', function (data) {
+  console.log('Event data', data);
+
+  // Set the state.
+  hasConversation = true;
+
+  // Load the Contact information based off of the event data. And set tab to Info.
+  loadContact(data.contact);
+  showInfo();
+});
+
+// Listen for the `no_conversation` event.  This can happen when opened to Inbox Zero.
+Front.on('no_conversation', function () {
+  console.log('No conversation');
+
+  // Set the state.
+  hasConversation = false;
+
+  // Display `No Contact` data and clear the notes and set the tab to Info.
+  displayContactInfo ("No Contact", "-");
+  displayCRMInfo("-", "-", "-");
```

Comment:
Instead of passing `"-"` manually, you can default the parameters in the method.